app/controllers/sessions_controller.rb

```diff
@@ -5,6 +5,10 @@ def create
     user = User.find_by email: params[:session][:email].downcase
     if user&.authenticate params[:session][:password]
       log_in user
+      case params[:session][:remember_me]
+      when "1" then remember user
```

Comment:
`"1"` khai báo vào settings
sao ko dùng toán tử ngôi số 3
` condition ? result_true : result_false`